site/plugins.json

```diff
@@ -621,5 +621,13 @@
     "package": "netlify-plugin-build-logger",
     "repo": "https://github.com/amar-p6/netlify-build-logger-plugin.git",
     "version": "1.0.3"
+  },
+  {
+    "author": "aaronbassett",
+    "description": "The New Relic Netlify plugin automatically notifies New Relic of Netlify build events and installs the New Relic browser agent.",
```

Comment:
Gentle suggestion: Maybe shorten description to "Automatically notifies New Relic of Netlify build events and installs the New Relic browser agent." 

This prevents the description from wrapping to a new line, leaving the word "agent" on its own